tests/statespace/test_CartesianProduct.cpp

```diff
@@ -20,7 +20,11 @@ TEST(CartesianProduct, Clone)
   for (auto i = 0u; i < 5u; ++i)
   {
     auto s1 = space.createState();
+#if DART_VERSION_AT_LEAST(6,7,0)
```

Comment:
I think this needs to merge changes from #480 first.

Reply:
Done!